### ISSUE TITLE
Unsupported major.minor version 51.0

### ISSUE THREAD

**Issue by bjoernalbers** [nroduit/weasis-pacs-connector#9]
I'm unable to deploy weasis-pacs-connector v6.1.1 on dcm4chee 2.18.3 psql.
After the restart I get this error in dcm4chee's `server.log`:

```
2017-08-24 12:11:09,660 ERROR -> (main) [org.jboss.deployment.MainDeployer] Could not create deployment: file:/opt/dcm4chee-2.18.3-psql/server/default/deploy/weasis-pacs-connecto
r.war
java.lang.UnsupportedClassVersionError: org/weasis/servlet/JnlpLauncher : Unsupported major.minor version 51.0
        at java.lang.ClassLoader.defineClass1(Native Method)
        at java.lang.ClassLoader.defineClassCond(ClassLoader.java:631)
        at java.lang.ClassLoader.defineClass(ClassLoader.java:615)
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:141)
        at java.net.URLClassLoader.defineClass(URLClassLoader.java:283)
        at java.net.URLClassLoader.access$000(URLClassLoader.java:58)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:197)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.net.URLClassLoader.findClass(URLClassLoader.java:190)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:306)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
        at org.jboss.wsf.container.jboss42.JAXWSDeployerHookPreJSE.getEndpointBeans(JAXWSDeployerHookPreJSE.java:145)
        at org.jboss.wsf.container.jboss42.JAXWSDeployerHookPreJSE.isWebServiceDeployment(JAXWSDeployerHookPreJSE.java:104)
        at org.jboss.wsf.container.jboss42.ArchiveDeployerHook.deploy(ArchiveDeployerHook.java:87)
        at org.jboss.wsf.container.jboss42.DeployerInterceptor.create(DeployerInterceptor.java:79)
        at org.jboss.deployment.SubDeployerInterceptorSupport$XMBeanInterceptor.create(SubDeployerInterceptorSupport.java:180)
        at org.jboss.deployment.SubDeployerInterceptor.invoke(SubDeployerInterceptor.java:91)
        at org.jboss.mx.server.Invocation.invoke(Invocation.java:88)
        at org.jboss.mx.server.AbstractMBeanInvoker.invoke(AbstractMBeanInvoker.java:264)
        at org.jboss.mx.server.MBeanServerImpl.invoke(MBeanServerImpl.java:659)
        at org.jboss.mx.util.MBeanProxyExt.invoke(MBeanProxyExt.java:210)
        at com.sun.proxy.$Proxy47.create(Unknown Source)
        at org.jboss.deployment.MainDeployer.create(MainDeployer.java:969)
        at org.jboss.deployment.MainDeployer.deploy(MainDeployer.java:818)
        at org.jboss.deployment.MainDeployer.deploy(MainDeployer.java:782)
        at sun.reflect.GeneratedMethodAccessor24.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
        at java.lang.reflect.Method.invoke(Method.java:597)
        at org.jboss.mx.interceptor.ReflectedDispatcher.invoke(ReflectedDispatcher.java:155)
        at org.jboss.mx.server.Invocation.dispatch(Invocation.java:94)
...
```

I didn't change the defaults and set only the `WebviewerNames` to weasis.

Thanks in advance!

**Comment by bjoernalbers** [comment]
PS: I've installed Weasis according to the [official instructions](https://dcm4che.atlassian.net/wiki/spaces/WEA/pages/3670023/Installing+Weasis+in+DCM4CHEE).
And I'm using Weasis 2.5.2 since 2.5.3 hasn't been localized yet.

**Comment by nroduit** [comment]
This means the container is running with Java 6. weasis-pacs-connector 6.x requires at least Java 7. I would also recommend to use the latest Java 7 for dcm4chee 2.18.3.

I've just published [weasis-pacs-connector 6.1.3](https://sourceforge.net/projects/dcm4che/files/Weasis/weasis-pacs-connector/).

Concerning the localization, it is a separate project with its own cycle (compliant from 2.x). So you can install 2.5.3 and either take weasis-i18n from the previous version or [build the latest](https://dcm4che.atlassian.net/wiki/spaces/WEA/pages/3670020/Translating+Weasis+Plug-ins).

**Comment by bjoernalbers** [comment]
Thanks!
